test/markup/clojure/globals_definition.txt

```diff
@@ -6,7 +6,7 @@
 (defn clojure-function [args]
   (let [string   "multiline\nstring"
         regexp   #"regexp"
-        number   100,000
+        number   100000
```

Comment:
Is this just invalid and it's wrong that we've always had it here or what?

Reply:
, is treated like a whitespace.
So this is actually 100 000, two separate numbers. (let) needs an even number of forms, with binding targets on the left.

There is no thousands separator in  Clojure.